src/components/Buttons.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from '../components/Buttons.scss';
+
+const ButtonCreator = (type) => {
```

Comment:
`ButtonCreator` is not a component so it should start with a lowecase letter.

(It's a function returning a component, but not a component itself. You can not do `<ButtonCreator/>`)